3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,68 @@
+
+const ApiKey = `https://open.er-api.com/v6/latest/EUR`;
+
+
+
+let toSelectList = document.createElement("select");
+let fromSelectList = document.createElement("select");
+let rates;
```

Comment:
Do you think you can move this declaration inside the function where `rates` is used?

In general, it is a good idea to try to avoid global variables as much as possible, since it becomes harder to control (in larger applications) what value they have at a given time.